### DISCUSSION TITLE
Back camera not working

### DISCUSSION THREAD

**Discussion by Sarangem** [community/community#86510] [GitHub Education]
### Hi everyone,

When I was applying and [uploading proof of academic status](https://education.github.com/discount_requests/13197442/additional_information) and chosen "camera2 0, facing back", it didn't switch to back camera of my phone. Pictures of front camera of ID was blurry. How can I switch to back camera? (I was using chrome, other two options were "camera2 1, facing front" and "camera2 2, facing back".

**Comment by Shahad-Alsubhi** [comment]
did you solve this problem? i'm facing it now

**Reply by Sarangem** [reply]
I took another phone and back camera was working there. I think it is a device-related problem.

**Comment by samus-aran** [comment]
Hi @ProneelPal & @Shahad-Alsubhi 👋 

We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Please ensure that you have allowed camera access on the browser you're using to apply.  The steps for this will vary depending on which browser you are using. 
2. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
3. If you can, print off your evidence and take a photo of this copy.
4. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).

If you are able to share what phone version & OS you're using and browser version we can improve our documentation.

**Reply by royess** [reply]
Have the same problem with my iPhone 12 mini (IOS 17.3.1) and iPad Pro (17.2)

**Comment by Todo56** [comment]
Same problem with my iPad and iPhone XR. Using safari and chrome browser.

**Reply by nihilisticzebra** [reply]
same here did you find a fix ?

**Reply by Todo56** [reply]
> same here did you find a fix ?

Basically take a picture of only the relevant stuff: your schools name, your full name, date of issue of the paper...

**Reply by nihilisticzebra** [reply]
i've tried that but get different rejection reasons each time

**Reply by Todo56** [reply]
> i've tried that but get different rejection reasons each time 

So what I did to solve it was I installed an app to make my phone a webcam. So instead of using directly my phone I controlled it through my desktop as a webcam and used that. I used the software "Elgato Camera Hub".

**Comment by h1ac** [comment]
I used the opera browser and It let me choose the back camera! I got accepted :)

**Comment by Cl9s4bl3** [comment]
Hi! I've got a working with a bit of a trick. (I used Firefox)

First, I selected what type of document I want to approve.
Then I took a selfie (doesn't really matter what's on it) and then BEFORE I selected "Reset", I chose the back camera. After this, I pressed on "Reset" and It worked, and I could take the picture. 2 minutes later, I've got approved.

**Reply by mcharrak** [reply]
This does not work for me. As soon as I took a selfie/image, it is not possible to change to a different camera anymore.

**Comment by dipan99** [comment]
This problem still exists. GitHub has made it extremely hard for students to apply for this. And this feels intentional. On most devices, the rear cam does not work. And front camera pictures are not very clear. Clearly there exist better methods for verification, but they don't want us to avail the benefits.
